contracts/v2.6.2/src/hello.cairo

```diff
@@ -2,6 +2,7 @@
 trait IHello<T> {
     fn get_name(self: @T) -> felt252;
     fn set_name(ref self: T, name: felt252);
+    fn sum(ref self: T, a: felt252, b: felt252) -> felt252;
```

Comment:
It should be a snapshot here instead of a ref.

Reply:
ops! sorry